docs/acton-by-example/src/types.md

```diff
@@ -2,6 +2,40 @@
 
 Acton is a statically typed language. Unlike traditional languages like C or Java, where the type of every variable must be explicitly stated, we can write most Acton programs without types. The Acton compiler features a powerful type inferencer which will then infer the type.
 
+To see the inferred type signatures after an Acton program is compiled you can use the `--sigs` option of the compiler. As rhe name suggests, this will print out the type signatures for the functions, actors and their attributes and methods found in the compiled file.
```

Comment:
I want to place less emphasis on the built / compiled part. It used to be that actonc would compile a source file no matter what but now we check if alternate output is enabled, such as --sigs, and then exit before doing final compilation. I think we still write a .ty file to disc. The point is that the focus is printing some output, not compiling, so rewording to reflect that...

```suggestion
To see the inferred types of an Acton program, use the `--sigs` option of the compiler. As the name suggests, this will print out the type signatures for functions, actors and their attributes and methods in the specified Acton module.
```